drivers/public_reports/app/models/public_reports/state_level_homelessness.rb

```diff
@@ -14,6 +14,14 @@ class StateLevelHomelessness < ::PublicReports::Report
     include Memery
     acts_as_paranoid
 
+    validate :validate_filter_dates_span_one_year, on: :create
+
+    def validate_filter_dates_span_one_year
+      return if ((filter_object.end - filter_object.start) / 365).round > 0
```

Comment:
Might be nicer to use ruby's date math:
```suggestion
      return if filter_object.start + 1.years >= filter_object.end
```

Do you think there's a reason not do do that?

Reply:
this makes sense to me, including it [here](https://github.com/greenriver/hmis-warehouse/pull/5114/commits/e77d06491460a127414ed0b39466e5ab9b8fd6df) with <= instead of >=